README.md

```diff
@@ -62,214 +61,73 @@ Group Project Navigator is always displayed and provides quick access to most co
 * Private discussions - connect with teammates.
 * Ask Teaching Assistant - ask course staff for help in seconds.
 
-## Setup
+## High Level Overview
 
-In order to use Group Project XBlock v2 in a course, XBlock must be installed into LMS environment and enabled in 
-course's advanced settings. Also, Group Project XBlock uses a number of instance-specific configuration variables, so 
-setting them is considered a part of setup process.
+This is covered in more detail in [the authoring documentation](/docs/authoring.md). 
 
-### Installing XBlock into LMS
+### Project XBlock 
 
-If you're using edx-solutions/edx-platform fork, Group Project v2 XBlock is already installed. Otherwise, installing 
-Group Project XBlock v2 is done the very same way other XBlocks are installed:
+Top most component of Group Work v2 is Project XBlock. Project XBlock holds a number of activities. 
+Workgroups are defined on project level, i.e. student work with the same people thought all activities. Workgroups 
```

Comment:
Rephrased it but differently than you suggested: 604d8ba77a15931a6d3f1571390fbe8a8b9477f1 see if you are happy with it.